scss/components/_button.scss

```diff
@@ -341,6 +341,32 @@ $button-transition: background-color 0.25s ease-out, color 0.25s ease-out !defau
       }
     }
 
+    // Clear style
+    @if $button-fill != clear {
+      &.clear {
+        @include button-clear;
+        @include button-clear-style;
+
+        @each $name, $color in $button-palette {
```

Comment:
@canterberrie Thanks for the merge 
but dear mam can you explain me a bit about your additions